docs/en/modules/somfy.md

```diff
@@ -0,0 +1,44 @@
+# Somfy module
+| Since  | Origin / Contributor  | Maintainer  | Source  |
+| :----- | :-------------------- | :---------- | :------ |
+| 2016-09-27 | [vsky279](https://github.com/vsky279) | [vsky279](https://github.com/vsky279) | [somfy.c](../../../app/modules/somfy.c)|
+
+This module provides a simple interface to control Somfy blinds via an RF transmitter (433.42 MHz). It is based on [Nickduino Somfy Remote Arduino skecth](https://github.com/Nickduino/Somfy_Remote). 
+
+The hardware used is the standard [433 MHz RF transmitter](http://www.ebay.com/sch/i.html?_from=R40&_trksid=p2050601.m570.l1313.TR0.TRC0.H0.X433Mhz+RF+transmitter+and+receiver+link+kit+for+Arduino%2FARM%2FMCU+WL.TRS0&_nkw=433Mhz+RF+transmitter+and+receiver+link+kit+for+Arduino%2FARM%2FMCU+WL&_sacat=0) (on eBay usually sold also with the receiver). Unfortunately these chips are usually transmitting at he frequency of 433.92MHz so the crystal resonator should be replaced with the [433.42 MHz resonator](http://www.ebay.com/sch/i.html?_from=R40&_trksid=p2047675.m570.l1313.TR0.TRC0.H0.X433.42MHz+Resonator+Crystals+DIP-3+TO-39.TRS0&_nkw=433.42MHz+Resonator+Crystals+DIP-3+TO-39&_sacat=0) though some reporting that it is working even with the original crystal.
```

Comment:
Ok. I guess there is no official data sheet. So I leave there just the "433 MHz RF transmitter". Everybody should be able to google it.